### ISSUE TITLE
Update readme to acknowledge that Entity Reference is now in core

### ISSUE THREAD

**Issue by stpaultim** [backdrop-contrib/references#18]
Is there any future for this module? Should we update the README to indicate that Entity Reference is now in core?

@jromine

**Comment by olafgrabienski** [comment]
> Is there any future for this module?

I agree, some indication to Entity Reference would be helpful, also if Entity Reference provides already the same features, e.g. regarding integration with other modules. And provide information if it's possible to switch to Entity Reference, or not yet.

**Comment by herbdool** [comment]
There's a PR for this https://github.com/backdrop-contrib/references/pull/20. @jromine. This may require the @backdrop-contrib/bug-squad

**Comment by olafgrabienski** [comment]
> There's a PR for this https://github.com/backdrop-contrib/references/pull/20.

Ah, good to know there is https://github.com/backdrop-contrib/entityreference_migration, and the PR mentions it.
